### ISSUE TITLE
"Run code" is not working 

### ISSUE THREAD

**Issue by alorne** [foundeo/cfdocs#1002]
I get the error "Self-suppression not permitted" when running any code on the Coldfusion 2018 option.
![image](https://user-images.githubusercontent.com/14872635/47446272-e73f2780-d788-11e8-8b8d-e7cdf257e4aa.png)

**Comment by pfreitag** [comment]
Yup I think that is an issue with trycf.com since I get that there on CF2018 as well, if you switch to CF2016 it should work. cc: @abramadams
